trin-core/src/ethereum/bridge.rs

```diff
@@ -0,0 +1,227 @@
+use async_trait::async_trait;
```

Comment:
Just curious as to your reasoning for creating a new directory for this? Why is it named `ethereum`? What else do you envision living inside this namespace?

Reply:
Right, so the idea is that this would also host our own type definitions for Ethereum data structures. One example for why these would be necessary is that the types defined in `ethers-core` do not implement the `rlp` serialization traits. We would need to create our own types for this. I'm open to other ways of organizing this, but I thought that the bridge would live alongside the types of data that it collects.